README.md

```diff
@@ -1628,17 +1629,33 @@ schema_data = [record.oRecordData for record in schema_records]
 index_records = client.command(ORIENTDB_INDEX_RECORDS_QUERY)
 index_query_data = [record.oRecordData for record in index_records]
 
+# Build SchemaGraph.
 schema_graph = get_orientdb_schema_graph(schema_data, index_query_data)
 
+# Get all the subclasses of a class.
 print(schema_graph.get_subclass_set('Animal'))
 # {'Animal', 'Dog'}
 
+# Get all the outgoing edge classes of a vertex class.
+print(schema_graph.get_vertex_schema_element_or_raise('Animal').out_connections)
+# {'Animal_Eats', 'Animal_FedAt', 'Animal_LivesIn'}
+
+# Get the vertex classes allowed in the arrow side of an edge class.
```

Comment:
I don't particularly like calling it "the arrow side" since that terminology feels imprecise. Perhaps `Get the vertex classes allowed as the terminal vertex of an edge class`? Not totally sure. https://cs.stackexchange.com/questions/70928/name-for-the-starting-vertex-and-ending-vertex-of-a-directed-edge

Reply:
Ok I addressed this comment. I changed it from out_connections to in_connections so that I could use the phrase the the "tail vertex" which I feel is as precise and clearer that "terminal vertex".